editorconfig-maven-plugin/src/main/java/org/ec4j/maven/AbstractEditorconfigMojo.java

```diff
@@ -129,6 +131,15 @@
     @Parameter(property = "editorconfig.failOnNoMatchingProperties", defaultValue = "true")
     protected boolean failOnNoMatchingProperties;
 
+    /**
+     * File containing exclude patterns to add to any existing exclude patterns. Empty lines and lines starting with #
+     * are ignored.
+     *
+     * @since 0.0.12
+     */
+    @Parameter(property = "editorconfig.excludesFile", readonly = true)
+    protected File excludesFile;
```

Comment:
I wonder whether this should be a list/array? Users might have several of those. (I have no strong opinion)

Reply:
I sort of based it on https://maven.apache.org/surefire/maven-surefire-plugin/test-mojo.html#excludesFile, but it could support more than one file if you want.

Reply:
OK, let's have just one file.